### ISSUE TITLE
HP Tuning with keras model and setting hyperparameterMetric to a Evaluation Metric and not training metric

### ISSUE THREAD

**Issue by datistiquo** [GoogleCloudPlatform/community#918]
Hi,

with estimators I can easily set the desired hyperparameterMetric to the proper metric on evaluation data. But I don't see how I can do that for a keras (tf.keras and keras) model?

I mean where can I "assign" the right metric? I need the hyperparameterMetric to be the metric for evaluation data.

**Comment by ToddKopriva** [comment]
The Issues section of this GitHub repository is not the best place to get such questions answered.

I recommend bringing your question to one of the forums described here:
https://cloud.google.com/support/docs/community

Many members of the GCP community---including Google employees and other experts---help out on those forums.

**Comment by datistiquo** [comment]
Thanks. I postet it now on stackoverflow. but maybe someone is also here:

model.fit returns a dict like:
```

{'acc': [0.9843952109499714],
 'loss': [0.050826362343496051],
 'val_acc': [0.98403786838658314],
 'val_loss': [0.0502210383056177]
}
```
Does GCP works now if I just set my desired validation metric to 'val_acc' in the config file without doing anything else?
